pkg/apis/resource/validation/validation.go

```diff
@@ -802,17 +802,24 @@ func validateNetworkDeviceData(networkDeviceData *resource.NetworkDeviceData, fl
 
 	allErrs = append(allErrs, validateSet(networkDeviceData.IPs, maxIPs,
 		func(address string, fldPath *field.Path) field.ErrorList {
-			return validation.IsValidCIDR(fldPath, address)
-		},
-		func(address string) (string, string) {
 			// reformat CIDR to handle different ways IPs can be written
 			// (e.g. 2001:db8::1/64 == 2001:0db8::1/64)
 			ip, ipNet, err := netutils.ParseCIDRSloppy(address)
 			if err != nil {
-				return "", "" // will fail at IsValidCIDR
+				// must fail
+				return validation.IsValidCIDR(fldPath, address)
```

Comment:
note to self, this just do the same , parse the cidr, but  throws the expected Errorlist
https://github.com/kubernetes/kubernetes/blob/21525f39e08bf680b457cab285d73f4e537a52da/staging/src/k8s.io/apimachinery/pkg/util/validation/validation.go#L402-L409